openmetadata-ui/src/main/resources/ui/src/axiosAPIs/feedsAPI.ts

```diff
@@ -26,6 +27,26 @@ export const getAllFeeds: Function = (
   });
 };
 
+export const getFeedsWithFilter: Function = (
+  userId?: string,
+  filterType?: FeedFilter
+): Promise<AxiosResponse> => {
+  let config = {};
+  if (typeof userId != 'undefined' && userId) {
```

Comment:
Use ` isUndefined` helper method from lodash.
```suggestion
  if (!isUndefined(userId)) {
```